modules/scratchFetch.ts

```diff
@@ -0,0 +1,218 @@
+import { defineEvent } from "strife.js";
+import fetch from "node-fetch";
+import { getSettings } from "./settings.js";
+
+defineEvent("messageCreate", async (message) => {
+	const notSet = getSettings(message.author, false)?.scratchEmbeds === undefined;
+	if (!getSettings(message.author, false).scratchEmbeds && !notSet) {
```

Comment:
For the second call of `getSettings`, just omit the `false` and the `&& !notSet`